graf2d/gpad/src/TPad.cxx

```diff
@@ -5854,9 +5868,14 @@ void TPad::SavePrimitive(std::ostream &out, Option_t * /*= ""*/)
          if (!strcmp(obj->GetName(),"Graph"))
             ((TGraph*)obj)->SetName(TString::Format("Graph%d",grnum++).Data());
       obj->SavePrimitive(out, (Option_t *)next.GetOption());
+      if (obj->InheritsFrom(TPad::Class())) {
+         if (opt.Contains("toplevel"))
+            out<<"   "<<pname<<"->cd();"<<std::endl;
+         else
+            out<<"   "<<cname<<"->cd();"<<std::endl;
+      }
    }
    out<<"   "<<cname<<"->Modified();"<<std::endl;
-   out<<"   "<<GetMother()->GetName()<<"->cd();"<<std::endl;
```

Comment:
You removed that line? is it intentional? if yes, are there any side effects?

Reply:
Yes, it was intentional since in the output C macro I need to write the (mother) pad name with `__X` suffix and `GetName()` would always return the original name. So I used other ways (including use of unused `option` to pass `toplevel` attribute to indicate certain pad to have customize actions) and this line is in principle not needed. It didn't break anything in my tests, and I found that it was actually redundant.